scripts/run_driver_tests.psm1

```diff
@@ -267,4 +267,43 @@ function Invoke-CICDStressTests
     popd
 }
 
+function Invoke-CICDPerformanceTests
+{
+    param(
+        [parameter(Mandatory = $true)][bool] $VerboseLogs,
+        [parameter(Mandatory = $true)][bool] $CaptureProfile)
+    Push-Location $WorkingDirectory
+
+    Write-Log "Executing eBPF kernel mode performance tests."
+
+    $LASTEXITCODE = 0
+
+    # Stop the services, remove the driver from verifier, and restart the services.
+    net.exe stop ebpfsvc
+    net.exe stop ebpfcore
+    # Remove the global verifier settings (this will remove the verifer interceptions that can degrade performance).
+    verifier.exe /volatile 0
+    # Remove the ebpfcore.sys driver from the verifier.
+    verifier.exe /volatile /removedriver ebpfcore.sys
+    net.exe start ebpfcore
+    net.exe start ebpfsvc
+
+    # Extract the performance test zip file.
+    Expand-Archive -Path .\bpf_performance.zip -DestinationPath .\bpf_performance -Force
```

Comment:
For my understanding: Doesn't the wpr addition add to the performance numbers? While it is good to have, it should not impact an increase in the numbers.

Reply:
Yes, but it's only for on-demand PR runs, which are used to investigate perf regressions.